publication-commons/src/test/java/no/unit/nva/publication/permission/strategy/ContributorPermissionStrategyTest.java

```diff
@@ -0,0 +1,79 @@
+package no.unit.nva.publication.permission.strategy;
+
+import static no.unit.nva.model.PublicationOperation.UNPUBLISH;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static no.unit.nva.testutils.RandomDataGenerator.randomUri;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import no.unit.nva.model.PublicationOperation;
+import no.unit.nva.model.associatedartifacts.AssociatedArtifactList;
+import no.unit.nva.model.role.Role;
+import no.unit.nva.publication.RequestUtil;
+import nva.commons.apigateway.exceptions.UnauthorizedException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+import org.junit.jupiter.params.provider.EnumSource.Mode;
+
+class ContributorPermissionStrategyTest extends PublicationPermissionStrategyTest {
+
+    @ParameterizedTest(name = "Should allow verified contributor {0} operation on non-degree resources")
+    @EnumSource(value = PublicationOperation.class, mode = Mode.EXCLUDE, names = {"DELETE", "TERMINATE",
```

Comment:
Better to `INCLUDE` operations instead of `EXCLUDE`? Easier to read the test, as one don't need to lookup the other operations.

Reply:
Applies to the test below as well

Reply:
I agree that the readability is worse, but it's done by design so that a change where something is added, it will make stuff fail unless you have taken explicit care of what the test should do. It's of course up for discussion if it's worth it or not.

Reply:
Ah, didn't think of it that way. Makes sense!